go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/google/go-cmp v0.5.6
 	github.com/google/uuid v1.1.2
 	github.com/googleapis/gnostic v0.5.5
+	github.com/kcp-dev/apimachinery v0.0.0-20220426201016-6bc8451bd9df
```

Comment:
This seems suspect? What caused this new import? Were we seriously not using these before?